src/CMakeLists.txt

```diff
@@ -944,7 +944,6 @@ install(PROGRAMS
 install(PROGRAMS
   ${CMAKE_SOURCE_DIR}/src/ceph-create-keys
 #  ${CMAKE_SOURCE_DIR}/src/ceph-disk
-  ${CMAKE_SOURCE_DIR}/src/ceph-disk-udev
```

Comment:
can we just remove `src/ceph-disk-udev` in our source tree, as it is not used anymore?